src/components/screens/DocsScreen/FeatureSnippets.tsx

```diff
@@ -13,14 +20,21 @@ export function PureFeatureSnippets({ framework, snippetsByFramework }) {
     }
   }
 
-  return (
+  return isInstallSnippet ? (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions
+    <div className="aside" onClick={() => logInstall(framework)}>
```

Comment:
This approach won't catch people who operate the Copy button with their keyboard. But that's likely done seldomly?